doc/decisions/error_message_format.md

```diff
@@ -0,0 +1,72 @@
+# Error message & handling concept
+
+## Problem
+
+Too verbose error message
+Currently for every error, 9 lines are shown in which most of them are not relevant to end users/administrators. One goal
+is to reduce the verbosity of such messages and let users/administrators see only information they need.
+
+## Constraints
+
+- Supporting multiple programming languages
+- Plugin System
+- Error Code should be preserved
+
+## Assumptions
+
+## Considered Alternatives
+
+Possible variations on what message should be displayed,
+eg. to keep the mountpoint information or on how wordings should be (with or without
+"Sorry, ...", coloring of certain parts of a message, etc.)
```

Comment:
Make a list of considered alternatives.

Reply:
The alternatives are pretty much endless on how you can rearrange an error message. I added some examples but I think it's is not feasible to list all possible variations of is possible for every field.

Reply:
The list of alternatives is always endless. It is about listing the *useful* but nevertheless discarded alternatives.